This patch removes the features=wasm from the dev-dependency on object,
it's not available in Debian and it doesn't seem needed for the tests
to pass at least on Linux.

Index: gimli/Cargo.toml
===================================================================
--- gimli.orig/Cargo.toml
+++ gimli/Cargo.toml
@@ -98,7 +98,6 @@ version = "1"
 
 [dev-dependencies.object]
 version = "0.30.0"
-features = ["wasm"]
 
 [dev-dependencies.rayon]
 version = "1.0"
